keanu-python/keanu/vertex/base.py

```diff
@@ -152,11 +152,11 @@ def __parse_args(args):
 
     @staticmethod
     def __parse_arg(arg):
-        if isinstance(arg, np.ndarray) or isinstance(arg, numbers.Number):
+        if isinstance(arg, const_arg_types):
```

Comment:
is `numbers.Number` the same as `(int, float, bool)`? I guess we don't need to talk about `numbers.Number` any more.